packages/shared/src/components/modals/common.tsx

```diff
@@ -85,6 +85,10 @@ const SearchReferralModal = dynamic(
     ),
 );
 
+const VideoModal = dynamic(
```

Comment:
I think we need to be more explicit with the filename and its component name, we can probably host another video in a modal, and this would make it confusing.

Reply:
I think that was the main idea why I made this modal not feature specific, but more generic with the main thing being hosting a video, with the idea that we might want to use it to host another video... 🤔 Do we not want this?

Reply:
I agree like the generic approach here :)

Reply:
Oh, then in that case, feel free to move forward if this was built to be generic 🚀